kbase-extension/static/kbase/js/widgets/appWidgets2/input/dynamicDropdownInput.js

```diff
@@ -157,9 +197,26 @@ define([
                         });
                         return results;
                     });
+            } else if (dataSource === 'search') {
+                if (call_params.constructor === Array){
+                    call_params = call_params[0]
+                }
+                return Promise.resolve(searchClient.search_objects(call_params))
+                    .then(function (results) {
+                        results.objects.forEach(function(obj, index) {
+                            obj = flattenObject(obj);
+                            obj.id = obj.guid;
+                            obj.text = obj[dd_options.selection_id];
+                            results.objects[index] = obj});
```

Comment:
It's hard to read where this block ends, the trailing `});`  should be on a separate line.